.travis.yml

```diff
@@ -11,22 +11,30 @@ matrix:
   allow_failures:
     - env: SHELL=zsh
 script:
-  - >
-    if ${LINT:-false}; then
-      shellcheck -s sh bin/shpec  # Just use th `sh` dialect
-    else
-      time $SHELL bin/shpec shpec/shpec_shpec.sh
-    fi
+  - |
+    ${LINT:-false} &&
+      exec shellcheck -s sh bin/shpec  # `sh` refers to POSIX
+    $SHELL bin/shpec shpec/shpec_shpec.sh
 cache: apt
 before_install:
-  - >
+  - |
+    get_shellcheck() {
+      (
+      for v 
```

Comment:
@hlangeveld - gee really? - so you don't need to pass explicit arguments to `for`?